serialzy/serializers/base_model.py

```diff
@@ -118,3 +123,50 @@ def __check_module_fits_serializer(self, typ: Type) -> bool:
                 if base_name == self.module:
                     return True
         return False
+
+    def serialize_with_meta(self, obj: Any, dest_dir: str, model_file_name: str) -> None:
+        """
+        :param obj: object to serialize into bytes
+        :param dest_dir: files with serialized model and meta are written into destination directory
+        :param model_file_name: serialized model's file name
+        :return: None
+        """
+        dest_dir_path = Path(dest_dir)
+
+        if not dest_dir_path.is_dir():
+            raise NotADirectoryError
```

Comment:
Here and downwards you a missing useful message like `f'given path {dest_dir_path} is not a dir'`